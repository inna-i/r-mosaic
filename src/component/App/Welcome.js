import React from 'react';

const Welcome = () => (
	<div className="welcome">
		<h1>Welcome to my App!</h1>
		<h3>Good to see you here</h3>
	</div>
);

export default Welcome;
