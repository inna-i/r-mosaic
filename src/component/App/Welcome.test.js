import React from 'react';
import { shallow } from 'enzyme';

import Welcome from './Welcome';

describe('Welcome component', () => {
	it('should render Welcome component', () => {
		// when
		const wrapper = shallow(<Welcome />);

		// then
		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
