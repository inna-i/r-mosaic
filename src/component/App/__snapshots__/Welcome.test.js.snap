// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component should render Welcome component 1`] = `
<div
  className="welcome"
>
  <h1>
    Welcome to my App!
  </h1>
  <h3>
    Good to see you here
  </h3>
</div>
`;
